tools/topology/topology2/platform/intel/deep-buffer.conf

```diff
@@ -46,3 +46,55 @@ Object.Base.route [
 		sink	gain.$DEEP_BUFFER_PIPELINE_ID.1
 	}
 ]
+
+# Spawn another instance
+IncludeByKey.DEEP_BUF_2 {
+"true"	{
+		Object.Pipeline.deepbuffer-playback [
+			{
+				index $DEEP_BUFFER_PIPELINE_ID_2
+
+				Object.Widget.host-copier.1 {
+					stream_name $DEEP_BUFFER_PCM_NAME_2
+					pcm_id $DEEP_BUFFER_PCM_ID_2
+				}
+				Object.Widget.gain.1 {
+					Object.Control.mixer.1 {
+						name 'Deepbuffer 2 Volume'
```

Comment:
probably best to find a different name with a parameter

Reply:
Ack